content/page/faq.md

```diff
@@ -0,0 +1,20 @@
+![Installing LinuxBoot on an Intel S2600 mainboard](images/s2600-flash.jpg)
+
+### Platform init and memory training
+As part of the hardware setup the so called "SoC" init is required. It consists
+of the platform bring-up and memory training. If the RAM training succeeds the
+Linux kernel can be loaded.
+
+### x86 bootblock magic
```

Comment:
bootblock is a coreboot term and here it's confusing: "what, disk?" Maybe we need another term.